### PR TITLE
PYSCAN-62 Run integration tests in CI

### PR DESCRIPTION
**Pull request by ghislainpiot**

### CODE REVIEW

File: its/tests/conftest.py [hunk 1]
```diff
@@ -38,17 +38,32 @@ def check_health(sonarqube_client: SonarQubeClient) -> bool:
...
+        while not check_health(sonarqube_client):
+            sleep(5)
+        response = sonarqube_client.get_system_status()
+        assert response.status_code == 200
+        status = response.json()["status"]
+        assert status == "UP"
+        return sonarqube_client
+else:
+    @pytest.fixture(scope="session")
+    def sonarqube_client(docker_ip, docker_services) -> SonarQubeClient:
```
Comment (joke1196) [inline its/tests/conftest.py:57]: Do you know why this does not work in the CI? As we do something similar in sonar-python.

File: .cirrus.yml [hunk 2]
```diff
@@ -101,12 +101,35 @@ build_task:
...
+  <<: *POETRY_TEMPLATE
+  its_script:
+    - unzip -q sonarqube_cache/sonarqube.zip -d sonarqube
+    - cd $(ls -d sonarqube/*/)
+    - ./bin/linux-x86-64/sonar.sh start
+    - cd -
+    - cd its/
+    - jfrog poetry-config --server-id-resolve repox --repo-resolve sonarsource-pypi
+    - jfrog poetry install
+    - while ! curl -X GET 'localhost:9000/' | grep "data-server-status=\"UP\""; do sleep 15; echo SLEEP;  done
```
Comment (joke1196) [inline .cirrus.yml:120]: Is this still useful as we are also waiting in the pytest fixture?

**Review by joke1196 — COMMENTED** [review]
Looks good to me. I just added a few comments/questions. Also don't forget to rebase from master.

File: its/tests/conftest.py (hunk 1 above)
Comment (ghislainpiot) [reply its/tests/conftest.py:57]: This removes the dependency on docker-compose, which I assumed would not be available in Cirrus

Comment (joke1196) [reply its/tests/conftest.py:58]: Ok I think for now this is ok but I would be good then to maybe get rid of the docker fixture and only have a fixture that would download and start SonarQube if needed.

**Review by joke1196 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**